src/watcher.rs

```diff
@@ -96,7 +96,18 @@ async fn run_debouncer<T: notify::Watcher>(
         for path in &opts.watch {
             watcher
                 .watch(path.as_std_path(), RecursiveMode::Recursive)
-                .into_diagnostic()?;
+                .map_err(|err| match err {
+                    notify::Error {
+                        kind: notify::ErrorKind::Io(e),
+                        ..
+                    } if e.kind() == std::io::ErrorKind::NotFound => {
+                        miette::MietteDiagnostic::new(format!(
+                            "Cannot watch path that doesn't exist: {:?}",
+                            path.absolute()
+                        ))
```

Comment:
(Needs `use miette::miette;` at the top of the file.)

```suggestion
                        miette!(
                            "Cannot watch path that doesn't exist: {:?}",
                            path.absolute()
                        )
```

Reply:
Done in 7bc0e60